credentials/generate-revocation-set.py

```diff
@@ -44,20 +45,61 @@ class RevocationType(Enum):
 TEST_NODE_URL_REST = "https://on.test-net.dcl.csa-iot.org"
 
 
-def use_dcld(dcld, production, cmdlist):
-    return [dcld] + cmdlist + (['--node', PRODUCTION_NODE_URL] if production else [])
-
-
 def extract_single_integer_attribute(subject, oid):
     attribute_list = subject.get_attributes_for_oid(oid)
 
     if len(attribute_list) == 1:
-        if attribute_list[0].value.isdigit():
-            return int(attribute_list[0].value)
+        return int(attribute_list[0].value, 16)
 
     return None
 
 
+class DCLDRequestsHelper:
+    def __init__(self, use_rest, dcld, production, rest_node_url):
```

Comment:
if python3, could we add some typing here? it would be nice to know which are bool and which are string and maybe even some info about them.

Also consider if we can improve on naming. Generally helper/support/util/common are filler words that should be avoided if possible.

What ius the purpose of the class? As a `helper` it feels like potentially a grab all of anything...

Reply:
`dcld` maybe should be renamed as `dcld_executable` or `dcld_binary`